src/website/apps/partners/migrations/0008_add_slugs_to_partners.py

```
@@ -0,0 +1,59 @@
+# Generated by Django 5.1.4 on 2025-09-20 15:57
+
+from django.db import migrations, models
+from django.utils.text import slugify
+
+
+def generate_partner_slugs(apps, schema_editor):
```

Ruff (0.13.1)

Unused function argument: `schema_editor`

(ARG001)